### PR TITLE
Announce releases on Discord

### PR DESCRIPTION
**Pull request by NotMyFault**
Addresses https://github.com/IntellectualSites/FastAsyncWorldEdit/issues/1971 for the FAVS part.

Render looks like:
Screenshot 2022-10-04 at 19 25 43

A PoC is available at https://github.com/IntellectualSites-Internal/fastasyncvoxelsniper-action-test/blob/main/.github/workflows/announce-release.yml

### CODE REVIEW

**Comment by NotMyFault** [comment]
I'd like some feedback from @IntellectualSites/core-team before turning this into a multipurpose organization workflow.

**Review by Aurelien30000 — COMMENTED** [review]
Automation is just fantastic :smile:

**Comment by SirYwell** [comment]
Good idea. Would it make sense to create a [composite action](https://docs.github.com/en/actions/creating-actions/creating-a-composite-action) to have one template for multiple repos? I'm not sure if the messages are similar enough for that, but if so, it would help with consistency.

**Comment by NotMyFault** [comment]
> Good idea. Would it make sense to create a [composite action](https://docs.github.com/en/actions/creating-actions/creating-a-composite-action) to have one template for multiple repos? I'm not sure if the messages are similar enough for that, but if so, it would help with consistency.

I considered an organization template for that yesterday, but given
- not every plugin we maintain is available on modrinth (yet?)
- all plugins require different role pings
- CurseForge subjects new submissions to approval, and therefore defaulting the URL makes no sense, if it's not available

which ultimately defeats the purpose of a general workflow, if you need to overwrite/inject almost every single line with different content, because the default doesn't match the plugin's case.

**Review by SirYwell — APPROVED** [review]
That's what I feared. Let's move forward with this for now then.

**Review by Aurelien30000 — APPROVED** [review]

**Comment by NotMyFault** [comment]
Re-requested the core team for review. If there's no negative feedback by next week Monday, I'll go ahead and apply this workflow to our projects.

File: .github/workflows/announce-release-on-discord.yml [hunk 1]
```diff
@@ -0,0 +1,26 @@
+name: Announce release on discord
+on:
+  release:
+    types: [published]
+
+jobs:
+  upload_asset:
```
Comment (NotMyFault) [inline .github/workflows/announce-release-on-discord.yml]:
```suggestion
  send_announcement:
```

**Pull request merged**